app/src/main/java/org/mozilla/fenix/collections/CreateCollectionFragment.kt

```diff
@@ -151,7 +151,7 @@ class CreateCollectionFragment : DialogFragment() {
                     dismiss()
                     viewLifecycleOwner.lifecycleScope.launch(Dispatchers.IO) {
                         requireComponents.core.tabCollectionStorage.renameCollection(it.collection, it.name)
```

Comment:
To be consistent with the change for `requireComponents.analytics`, should this become `context?.components?.core`?

Reply:
Yes, I agree. This definitely seems required if the latter is crashing.